pilot/pkg/networking/v1alpha3/cluster.go

```diff
@@ -44,14 +56,15 @@ func BuildClusters(env model.Environment, proxy model.Proxy) []*v2.Cluster {
 			return nil
 		}
 
-		clusters = append(clusters, buildInboundClusters(instances)...)
+		clusters = append(clusters, buildInboundClusters(env, instances)...)
 
+		// TODO: Bug? why only for sidecars?
```

Comment:
routers won't have co-located instances.

Reply:
No.. this is for the JWKURI thing.. not for proxying